packages/fast-components-react-base/src/dialog/dialog.props.ts

```diff
@@ -45,6 +45,14 @@ export interface DialogHandledProps extends DialogManagedClasses {
      */
     modal?: boolean;
 
+    /**
+     * Defines where focus goes when the dialog is unmounted.
+     * This is useful because modal dialogs will take focus back until that point in their life cycle.
+     * The element or ref provided will be focused when the dialog unmounts.
+     * Alternatively, providing a callback function gives developers more flexibility if needed.
+     */
+    refocusTarget?: React.RefObject<any> | HTMLElement | (() => void);
```

Comment:
What's the reason the Ref.RefObject can be `any`? Don't we want that to be an HTMLElement?

Reply:
Updated, but moved to it's own pr
https://github.com/microsoft/fast-dna/pull/2613